src/app/content/utils/seoUtils.ts

```diff
@@ -0,0 +1,20 @@
+import { assertDefined } from '../../utils';
+import { Book, Page } from '../types';
+import { archiveTreeSectionIsChapter, findArchiveTreeNode, getArchiveTreeSectionNumber } from './archiveTreeUtils';
+
+export const createTitle = (page: Page, book: Book): string => {
+  let node = assertDefined(findArchiveTreeNode(book.tree, page.id), `couldn't find node for a page id: ${page.id}`);
+  const domNode = new DOMParser().parseFromString(node.title, 'text/html');
+  const osText = domNode.querySelector('.os-text');
+  const nodeTitle = osText ? osText.textContent : page.title;
```

Comment:
I did it this way because I'm not sure if it is correct to assume that `node.title` will always have `os-text` but I think it might be the case here. cc @TomWoodward

Reply:
`splitTitleParts` already defaults to the whole string if `.os-title` is not found, you want to just use that. 

if you want you can make an alias that takes the title portion of the return value, like `getArchiveTreeSectionNumber` does